ext/mysqli/tests/bug42548.phpt

```diff
@@ -4,9 +4,8 @@ Bug #42548 PROCEDURE xxx can't return a result set in the given context (works i
 mysqli
 --SKIPIF--
 <?php
-require_once('skipifconnectfailure.inc');
-require_once('connect.inc');
-if (!$link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket)) {
+require_once 'connect.inc';
+if (!$link = @my_mysqli_connect($host, $user, $passwd, $db, $port, $socket)) {
     die(sprintf('skip Cannot connect to MySQL, [%d] %s.', mysqli_connect_errno(), mysqli_connect_error()));
 }
 if (mysqli_get_server_version($link) <= 50000) {
```

Comment:
Do we still support MySQL < 5.0.0?

Reply:
I don't know. I know we don't support libmysql < 5.1 but I am not sure about the server itself.

Reply:
Well, not really related to this PR anyway. (And we would need to be careful wrt. MySQL forks/clones.)